server/src/infra/migrations/1690140002097-Panoramas.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Panoramas1690140002097 implements MigrationInterface {
+  name = 'Panoramas1690140002097';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "assets" ADD "projectionType" character varying NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "exif" ADD "projectionType" character varying`);
+  }
```

Comment:
The annotation for `projectionType` is `@Column({ type: 'enum', enum: ProjectionType, enumName: 'projectionType', nullable: true })`. So this migration looks outdated